.gitignore

```diff
@@ -1,3 +1,4 @@
 info.plist
 persist
-node_modules
\ No newline at end of file
+node_modules
+config.js
```

Comment:
- Should not commit `config.js` 
- Create a sample to be cloned: `config.sample.js`